.projenrc.js

```diff
@@ -85,6 +85,18 @@ const installHelm = project.addTask('install-helm', {
 
 project.testTask.prependSpawn(installHelm);
 
+// test compatibility with constructs v10
+const compatTask = project.addTask('test:compat');
+compatTask.exec('cp package.json package.json.bak');
+compatTask.exec('yarn add --dev constructs@^10');
+compatTask.exec('npm list --depth=0 constructs --json'); // for debugging
```

Comment:
grep the version number so this command will fail in case your expectation failed